app/assets/scripts/components/dashboard/details-card.js

```diff
@@ -20,6 +22,19 @@ export default function DetailsCard({ measurements, date, coords }) {
               {formatThousands(measurements)}
             </HighlightText>
             <CardSubtitle className="card__subtitle">Measurements</CardSubtitle>
+
+            {sources &&
+              sources
+                .filter(s => s.readme)
+                .map(s => (
+                  <SmartLink
+                    key={s.id}
+                    to={`${config.api}/${s.readme}`}
+                    title="Suggest a new source"
+                  >
+                    {`${s.name} Technical Readme`}
```

Comment:
The displayed `source.name` feels like a duplication from the source displayed in the sources-card. Maybe label it with Technical Readme only? 
```suggestion
                    Technical Readme
```

Reply:
@AliceR I agree it is verbose but i think there is a possibility of having multiple sources and therefore multiple readme's (although as of now perhaps that does not exist). what do you think?

Reply:
I don't know enough about the concept of technical readme's to answer that. Maybe @rwegener2 can help?

Reply:
I agree with @jvntf that for now we should assume that we could have multiple sources and readmes.  We might change that in response to EDF's feedback next week but for now I think it the path that makes the most sense.